plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -616,9 +510,18 @@ public void testTimestamp(ZoneId sessionZone)
                 // time gap in Kathmandu
                 .addRoundTrip("timestamp(3)", "TIMESTAMP '1986-01-01 00:13:07.000'", TIMESTAMP_MILLIS, "TIMESTAMP '1986-01-01 00:13:07.000'")
 
-                .execute(getQueryRunner(), session, trinoCreateAsSelect(session, "test_timestamp"))
-                .execute(getQueryRunner(), session, trinoCreateAsSelect(getSession(), "test_timestamp"))
-                .execute(getQueryRunner(), session, trinoCreateAndInsert(session, "test_timestamp"));
+                .execute(getQueryRunner(), session, setup);
```

Comment:
I don't think this is equivalent. The 1st and 3rd cases are covered but the 2nd one is not (testing inserts from a different session and reads from a different session).

Reply:
Oh, I missed that, oops. I'll put it back with the others.